05/11/2017 19:33:47  (Database : isRunning())  com.mysql.jdbc.exceptions.jdbc4.MySQLNonTransientConnectionException: Could not create connection to database server. Attempted reconnect 3 times. Giving up.


06/11/2017 19:45:08  (Database : isRunning())  com.mysql.jdbc.exceptions.jdbc4.MySQLNonTransientConnectionException: Could not create connection to database server. Attempted reconnect 3 times. Giving up.


06/11/2017 19:58:32  (Database : isRunning())  com.mysql.jdbc.exceptions.jdbc4.MySQLNonTransientConnectionException: Could not create connection to database server. Attempted reconnect 3 times. Giving up.


06/11/2017 19:58:34  (Database : isRunning())  com.mysql.jdbc.exceptions.jdbc4.MySQLNonTransientConnectionException: Could not create connection to database server. Attempted reconnect 3 times. Giving up.


07/11/2017 19:49:13  (UserDatabase : getUsernames())  com.mysql.jdbc.exceptions.jdbc4.MySQLNonTransientConnectionException: Could not create connection to database server. Attempted reconnect 3 times. Giving up.


08/11/2017 19:35:48  (Database : isRunning())  com.mysql.jdbc.exceptions.jdbc4.MySQLNonTransientConnectionException: Could not create connection to database server. Attempted reconnect 3 times. Giving up.


08/11/2017 19:52:06  (Engine : execute())  java.util.InputMismatchException


08/11/2017 19:52:31  (Engine : execute())  java.util.InputMismatchException


08/11/2017 20:02:10  (Database : isRunning())  com.mysql.jdbc.exceptions.jdbc4.MySQLNonTransientConnectionException: Could not create connection to database server. Attempted reconnect 3 times. Giving up.


09/11/2017 18:45:53  (UserDatabase : getUsernames())  com.mysql.jdbc.exceptions.jdbc4.MySQLNonTransientConnectionException: Could not create connection to database server. Attempted reconnect 3 times. Giving up.


09/11/2017 18:52:26  (Database : isRunning())  com.mysql.jdbc.exceptions.jdbc4.MySQLNonTransientConnectionException: Could not create connection to database server. Attempted reconnect 3 times. Giving up.


09/11/2017 19:00:43  (Database : isRunning())  com.mysql.jdbc.exceptions.jdbc4.MySQLNonTransientConnectionException: Could not create connection to database server. Attempted reconnect 3 times. Giving up.


09/11/2017 19:00:44  (Database : isRunning())  com.mysql.jdbc.exceptions.jdbc4.MySQLNonTransientConnectionException: Could not create connection to database server. Attempted reconnect 3 times. Giving up.


10/11/2017 15:01:44  (Database : isRunning())  com.mysql.jdbc.exceptions.jdbc4.MySQLNonTransientConnectionException: Could not create connection to database server. Attempted reconnect 3 times. Giving up.


10/11/2017 15:42:31  (UserDatabase : getUserByID())  com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'didi' in 'where clause'


10/11/2017 16:05:28  (UserDatabase : giveUserPrivileges())  com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''manager'=true WHERE id=21' at line 1


10/11/2017 17:07:31  (UserDatabase : deleteUser())  com.mysql.jdbc.exceptions.jdbc4.MySQLNonTransientConnectionException: No operations allowed after connection closed.


10/11/2017 17:26:05  (Engine : execute())  java.lang.NullPointerException


10/11/2017 17:26:31  (Engine : execute())  java.lang.NullPointerException


10/11/2017 17:32:14  (Engine : execute())  java.lang.NullPointerException


10/11/2017 20:14:33  (UserDatabase : getUserByID())  java.lang.NullPointerException


10/11/2017 20:14:48  (UserDatabase : getUserByID())  java.lang.NullPointerException


10/11/2017 20:14:53  (UserDatabase : getUserByID())  java.lang.NullPointerException


10/11/2017 20:15:57  (UserDatabase : getUserByID())  java.lang.NullPointerException


11/11/2017 14:41:03  (UserDatabase : getUsernames())  com.mysql.jdbc.exceptions.jdbc4.MySQLNonTransientConnectionException: Could not create connection to database server. Attempted reconnect 3 times. Giving up.


